tests/limiter/limiter.js

```diff
@@ -182,14 +183,28 @@ describe('tests/limiter/limiter.js >', () => {
     });
 
     it('First user, 429', (done) => {
+      tu.toggleOverride('enableLimiterActivityLogs', true);
+      logger.on('logging', testLogMessage);
```

Comment:
It would be better to add this in a new "describe" specifically for logging to keep it separate from the functionality tests.

Reply:
done